README.md

```diff
@@ -1,19 +1,19 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
-> Description the project.
+> Add linters to html/css.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML/CSS
+- Node.js
+
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](127.0.0.1:5500/index.html)
 
 
 ## Getting Started
```

Comment:
I suggest you edit this section so you can add instructions on how to get a local copy of this project and how to install the necessary dependencies for it to run properly.